### PR TITLE
Search page revamp

### PR DESCRIPTION
**Pull request by mrvanagas**
Comepletely remade the search page. Instead of showing only 1 city forecast, the app is now able to show multiple cities forecast and show them at the same time:
image

The information is being stored and passed to the component via Redux.

Note: Proper styling will be done in a separate PR, which will overhaul the entire app.

### CODE REVIEW

File: client/src/Components/Card/Card.jsx [hunk 1]
```diff
@@ -3,31 +3,31 @@ import Spinner from '../Spinner/Spinner';
 import * as Style from './Card.styles';
 
 const Currentweather = ({ data }) => {
-  const weatherData = data[0];
+  const [weatherData] = data;
+  console.log(weatherData);
```
Comment (JuliaShavyrova) [inline client/src/Components/Card/Card.jsx]: please don't forget to remove console logs

File: client/src/Components/MultiCard/MultiCard.jsx [hunk 2]
```diff
@@ -2,21 +2,33 @@ import * as Style from './MultiCard.styles';
 
 import dateToWeekdayHelper from '../../Utils/dateHelper';
 
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { CITY_REMOVE } from '../../store/actionType';
 
 const MultiWeather = () => {
-  const data = useSelector(state => state.selectForecast.citiesForecast);
+  const citiesAndWeatherInfoFromRedux = useSelector(state => state.savedCitiesReducer.savedCities);
```
Comment (JuliaShavyrova) [inline client/src/Components/MultiCard/MultiCard.jsx]: no need to define that data comes from redux :)

Comment (mrvanagas) [reply client/src/Components/MultiCard/MultiCard.jsx]: Fixed

File: client/src/Components/Card/Card.jsx (hunk 1 above)
Comment (mrvanagas) [reply client/src/Components/Card/Card.jsx]: Done. Will make another pass for unneeded console.log in further PRs

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**